examples/makefile

```diff
@@ -105,6 +105,7 @@ clean-exec:
 	@rm -rf Example5* Example9* Example15* Example16*
 	@rm -f sphere_refined.* sol.* sol_u.* sol_p.*
 	@rm -f ex9.mesh ex9-mesh.* ex9-init.* ex9-final.*
+	@rm -f ex16.dat ex16p.dat gnuplot_ex16.inp gnuplot_ex16p.inp
```

Comment:
Are these really necessary?

Reply:
They must be left-overs from an outdated makefile.  I'll remove them.